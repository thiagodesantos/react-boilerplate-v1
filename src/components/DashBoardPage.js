import React from 'react';

const DashBoardPage = () => (
    <div>
        Pagina do dashboard
    </div>
);

export default DashBoardPage;