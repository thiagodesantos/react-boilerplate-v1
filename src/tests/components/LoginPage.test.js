import React from 'react';
import { shallow } from 'enzyme';
import { LoginPage } from '../../components/LoginPage';

test('Deveria renderizar o Login corretamente', () => {
    const wrapper = shallow( <LoginPage />);
    expect(wrapper).toMatchSnapshot();
});

test('Deveria chamar o startLogin no click do botÃ£o', () => {
    const startLogin = jest.fn();
    const wrapper = shallow(<LoginPage startLogin={startLogin}/>);
    wrapper.find('button').simulate('click');
    expect(startLogin).toHaveBeenCalled();
});
