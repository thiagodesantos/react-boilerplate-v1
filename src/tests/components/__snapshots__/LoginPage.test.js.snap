// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deveria renderizar o Login corretamente 1`] = `
<div
  className="box-layout"
>
  <div
    className="box-layout__box"
  >
    <h1
      className="box-layout__title"
    >
      BoilerPlate
    </h1>
    <p>
      Tag line for app!
    </p>
    <button
      className="button"
    >
      Login com o Google
    </button>
  </div>
</div>
`;
